driver/src/main.rs

```diff
@@ -68,9 +68,12 @@ struct Options {
     #[structopt(short = "i", long, env = "NETWORK_ID")]
     network_id: u64,
 
-    /// Which style of solver to use. Can be one of: 'NAIVE' for the naive
-    /// solver; 'MIP' for mixed integer programming solver; 'NLP' for non-linear
-    /// programming solver.
+    /// Which style of solver to use. Can be one of:
```

Comment:
ubernit: Not sure this comment is valuable here as it might easily be forgotten when a new flavor is added. I think the SolverType itself is documentation enough about the different solver types.

Reply:
Personally, I think these comments should stay since they are used to generate commandline help when running `driver --help`. But it definitely can be abridged as you mentioned to not exhaustively include all the types.